docs/index.md

```diff
@@ -34,13 +34,13 @@ Although in the FTC community dead wheels and odometry are often used synonymous
 
-So you ask yourself, **why would I choose the three wheel configuration?** "That's another module that I have to make and increases cost by ~30%!" At the time of writing, the three-wheel configuration is the most accurate option. This is because the Rev Expansion Hub utilizes a slow I2C implementation which results in 7ms per I2C call. The BNO055 IMU onboard the Rev Expansion Hub communicates via I2C. This will contribute a signficant amount of delay to your loop times which reduces integration accuracy (the 7ms is in addition to the 3ms call for all three dead wheels assuming bulk reads, so 10ms). Thus to calculate the heading with a two wheel configuration, it must call on this slow IMU. A three wheel configuration calculates heading via the two parallel wheels which can be bulk read. Thus, all three of your dead wheels can be read in 3ms (assuming bulk reads and that all your encoders are on one Rev Expansion/Control Hub). This has been seen to improve accuracy and reduce drift significantly. The upcoming Control Hubs, which half the I/O latency, may reduce this difference.
+So you ask yourself, **why would I choose the three wheel configuration?** "That's another module that I have to make and increases cost by ~30%!" At the time of writing, the three-wheel configuration is the most accurate option. This is because the Rev Expansion/Control Hub utilizes a slow I2C implementation which results in about 7ms per I2C call. The BNO055 IMU onboard the Rev Expansion/Control Hub communicates via I2C. This will contribute a signficant amount of delay to your loop times which reduces integration accuracy (the 7ms is in addition to the 3ms call for all dead wheels assuming bulk reads, so 10ms). To calculate the heading with a two-wheel configuration, it must call on this slow IMU for heading. A three-wheel configuration calculates heading via the two parallel wheels, which eliminates the 7ms I2C call. Since all three of your dead wheels can be read in 3ms (assuming bulk reads and that all your encoders are on one Rev Expansion/Control Hub), the loop time is greatly reduced. This has been seen to improve accuracy and reduce drift significantly. The upcoming Control Hubs, which perform the I/O at a 3x faster rate than the Expansion Hub, may reduce this difference between the two set-ups.
```

Comment:
I intentionally left out any comment about the Control Hub because it isn't set in stone yet.

Reply:
Which mention in that section?

Reply:
"The upcoming Control Hubs, which perform the I/O at a 3x faster rate than the Expansion Hub, may reduce this difference between the two set-ups."

Reply:
Gonna wait until Control Hub officially comes out and there are actual benchmarks to back it rather than anecdotal Discord talk :P

Reply:
I believe that NPE confirmed the 3x faster I/O, as well as Dansman through his own tests.

Reply:
Okay then